modules/nw-configuring-clb-timeouts.adoc

```diff
@@ -0,0 +1,42 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-clb-timeouts_{context}"]
+= Configuring Classic Load Balancer timeouts
+
+You can configure the default timeouts for a Classic Load Balancer (CLB) to extend idle connections.
+
+.Prerequisites
+
+* You need a deployed Ingress Controller on a running cluster.
+
+.Procedure
+
+* Using the `oc patch` command, set an AWS connection idle timeout of five minutes for the default `ingresscontroller`:
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Rephrasing option:


> If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to "run" or "enter" the command. In most cases, use one of the following patterns to introduce the code block:
>
>  -  \<Step description>, run the following command:
>
>  -  \<Step description>, enter the following command: